client/src/components/event-types/all-event-types/EventType.vue

```diff
@@ -10,18 +10,21 @@
         <div class="event-type-content">
             <h3>{{ eventType.name }}</h3>
             <span class="event-type-about"
-                >{{ eventType.duration }} {{ lang.MINS }}, {{ eventType.type }}
-            </span>
+                >{{ eventType.duration }} {{ lang.MINS }},
+                {{ eventType.type }}</span
+            >
         </div>
         <div class="event-type-invitee mt-9 mb-2">
             <Avatar :size="24" :color="'black'"></Avatar>
         </div>
         <VDivider />
         <VRow class="event-type-actions">
             <VCol class="duration text-left" cols="4">
-                <RouterLink :to="{ name: 'PublicEvent' }">
-                    <span>/{{ eventType.slug }}</span></RouterLink
+                <RouterLink
+                    :to="{ path: `/${eventType.owner.name}/${eventType.id}` }"
```

Comment:
`owner.nickname`, we will with owner nickname
`eventType.slug`
`/eventType.owner.nickname/eventType.slug`

Reply:
I think we can fix it later, it's probably now necessary right now

Reply:
I think it is necessary. The owner's name is not unique and can contain inappropriate symbols for the URL.